apps/views.py

```diff
@@ -72,8 +72,13 @@ def post(self, request, project, app_slug, app_id):
         project = self.get_project(project, post=True)
         instance = self.get_instance(app_slug, app_id, post=True)
 
-        # container name is often same as subdomain name
-        container = instance.subdomain.subdomain
+        # get container name from UI (subdomain or copy-to-pvc) if none exists then use subdomain name
+        container = request.POST.get("container", "") or instance.subdomain.subdomain
+
+        # Perform data validation
+        if not container.isalnum() and container != "":
```

Comment:
This will not work for containers with names containing dashes such as "abc-def"

Reply:
It could probably use SubdomainCandidateName.validate_subdomain() for consistency.